src/model/document/document.ts

```diff
@@ -219,12 +260,14 @@ export function useDocument<TElements extends QuantumDocumentElementTypes<readon
   const elementList = useElementList()
   const elementSelection = useElementSelection()
   const elementFocus = useElementFocus()
+  const elementDrag = useElementDrag()
 
   const rootScope = createElement(ScopeElementType.typeName, {
     position: Vector2.zero,
   }) // TODO: Scope size:
 
   function createElement<T extends keyof TElements>(typeName: T, options: QuantumElementCreationOptions): QReturnType<TElements[T]['elementType']> {
+    console.log('new')
```

Comment:
I guess this was there for debugging and not meant to be merged? 
I can't count the number of times I've left in a console.log in something for production though 😆

Reply:
whoops!